app/src/main/res/xml/pref_advanced_settings.xml

```diff
@@ -902,6 +902,17 @@
                 android:summary="@string/summary_write_Battery_Information_for_additional_analytic"
                 android:title="@string/title_Batterylog" />
         </PreferenceScreen>
+
+        <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+            android:key="xdrip_libre2_advanced_settings"
+            android:title="Advanced settings for Libre 2">
+            <SwitchPreference
+                android:defaultValue="false"
+                android:key="Libre2_showRawGraph"
+                android:summary="switch on to show the raw values on the Graph"
```

Comment:
Those strings should be extracted to strings.xml and referenced to facilitate localisation.

Reply:
Will be done later, as Jamorham told me in the past to put those strings to translation after the new functionality will be ok and also used (like 4 weeks after implementation), I have them on my list to switch them later...